pkgs/modules/default.nix

```diff
@@ -12,16 +12,16 @@ let
 
   modulesList = [
     (import ./python {
-      python = pkgs-23_05.python38Full;
```

Comment:
won't 3.8 still fail to build `python38Packages` because sphinx is unsupported now? I think we're stuck unless we try to patch

Reply:
It worked for me. CI will test the build too.

Reply:
Huh. OK!